src/ray/raylet/local_object_manager.cc

```diff
@@ -228,6 +235,9 @@ void LocalObjectManager::SpillObjectsInternal(
         for (const auto &object_id : objects_to_spill) {
           RAY_LOG(DEBUG) << "Sending spill request for object " << object_id;
           request.add_object_ids_to_spill(object_id.Binary());
+          auto it = objects_pending_spill_.find(object_id);
```

Comment:
```suggestion
          RAY_CHECK(objects_pending_spill_.count(object_id) > 0);
```

Reply:
We need to get the owner address value from the iterator, so I think that the `it != objects_pending_spill_.end()` check is more prudent.

Reply:
Makes sense!